### PR TITLE
Expose onScroll event

### PR DESCRIPTION
**Pull request by averted**
If you pass onScroll to ParallaxView as a prop it gets overwritten at component level (there is already onScroll that handles parallax animation).

This allows you to extend it.

### CODE REVIEW

**Comment by ericraio** [comment]
+1

**Comment by lelandrichardson** [comment]
I know this PR was submitted like "forever" ago, but I'm going to be making a couple of updates to this module.

Sorry about that!  Anyway, I like the idea of this change, but i'm not sure I like putting an `Animated.event` call into each handleScroll invocation like this. Animated.event is really only meant to be called once, and is more expensive than if we just did the manual mapping here.

Alternatively, we could call `Animated.event` once (in render) and then pass in a composition of that function and the onScroll prop passed in, and pass that into the scrollview.

Basically, I don't want `Animated.event` to get called every 16ms...  Can you change this so that it wont?  If not, I'll try and make the change myself in a bit.  Let me know, thanks.

**Comment by TroySK** [comment]
Any update on this? How do we access the onScroll event?

**Comment by yeegr** [comment]
July 12, seems still bloked

**Comment by ericraio** [comment]
anyone have a fix??

**Comment by lelandrichardson** [comment]
This is now fixed in 2.0.6
